README.md

```diff
@@ -166,15 +166,23 @@ mode or the command line mode.
 
 There are many bugs and TODOs.
 
-## Installation
+Plotman will always look in the current directory for the `config.yaml` file
+([see also](https://github.com/ericaltendorf/plotman/pull/61#issuecomment-812967363)).
 
-This program requires `psutil`, `pyfakefs`, `pyyaml` and `texttable`.
+## Installation
 
 Installation for Ubuntu 20.04:
```

Comment:
The only bit even Linux specific at this point is the use of `.` for the activation.  Though it is probably just as well to say Linux since we don't support Windows presently.

```suggestion
Installation for Linux:
```